pkg/task/meta.go

```diff
@@ -314,14 +319,14 @@ func (m *metaTaskExecutor) asRollbackInstance(objectName string) (*metaTaskExecu
 		return nil, true, err
 	}
 
```

Comment:
Maybe a comment why the below code has been commented would help.

Reply:
OK. Will do that.